data-pipeline-ffi/src/trace_exporter.rs

```diff
@@ -215,6 +223,30 @@ pub unsafe extern "C" fn ddog_trace_exporter_config_set_service(
     }
 }
 
+/// Enables metrics.
+#[no_mangle]
+pub unsafe extern "C" fn ddog_trace_exporter_config_enable_telemetry(
```

Comment:
/nit

I would prefer staying consistent with other APIs and less cognitive load

```suggestion
pub unsafe extern "C" fn ddog_trace_exporter_config_set_telemetry(
```

Reply:
+1 to what @VianneyRuhlmann said. Consistency is important, but I think there is a distinction between "setting" a param and "enabling" functionality.